### PR TITLE
Support running the validator client without a beacon node

### PR DESCRIPTION
**Pull request by ajsutton**
Adds two new options:
 * `teku_beacon_enabled` which controls whether the beacon node is deployed or not. Setting to False deploys the validator client without a local beacon node
 * `teku_beacon_node_api_endpoints` option that allows specifying the beacon node endpoints for the validator client to use.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -103,7 +103,8 @@ teku_log_file: "{{ teku_log_dir }}/teku.log"
 teku_config_file: "{{ teku_config_dir }}/teku.yml"
 
 # determine whether to run as standalone validator client. Default is not to run standalone validator
-teku_standalone_validator: False
+teku_beacon_enabled: True
+teku_standalone_validator: "{{ not teku_beacon_enabled | bool }}"
```
Comment (rolfyone) [inline defaults/main.yml:107]: we may need to be able to set `submit-early-attestations` (its called something like that) if we're using standalone validators too... one use-case being the validators that run against infura...

**Review by rolfyone — APPROVED** [review]
LGTM, but we may need an extra field to make this work in some situations.

File: README.md [hunk 2]
```diff
@@ -113,7 +114,9 @@ List of variables which are not defined with default values in ansible role. How
 | `teku_validators_graffiti`                        | `validators-graffiti`                        | Graffiti to include during block creation (gets converted to bytes and padded to Bytes32)   |
 | `teku_validators_keystore_locking_enabled`        | `validators-keystore-locking-enabled`        | Enable locking validator keystore files (Valid values True, False)                          |
 | `teku_validators_performance_tracking_enabled`    | `validators-performance-tracking-enabled`    | Enable validator performance tracking and logging (Valid values True, False)                |
+| `teku_validators_early_attestations_enabled`      | `validators-early-attestations-enabled`      | Enable early attestation production (Valid values True, False)                              |
```
Comment (joshuafernandes) [inline README.md:117]: Should the second column here be true/false?

File: tasks/config.yml [hunk 3]
```diff
@@ -24,7 +24,23 @@
       group: "{{ teku_group }}"
     become: true
     register: config_toml
-  when: "not teku_standalone_validator | bool"
+  when:
+    - "not teku_standalone_validator | bool"
```
Comment (joshuafernandes) [inline tasks/config.yml:28]: more of a nit - I prefer the old school (a) and (b) so its really clear this is an `and` operation

File: README.md (hunk 2 above)
Comment (ajsutton) [reply README.md:117]: Second column is what the config file item name is.

**Pull request merged**